### ISSUE TITLE
25/75 module requiring headers

### ISSUE THREAD

**Issue by tnferrell** [cfpb/cfgov-refresh#1252]
25/75 organism is now requiring a group header when that should be optional. @kurtw @kave

**Comment by kurtw** [comment]
That _should_ be in a PR right now but I'm not near my laptop to check
easily.

**Comment by kave** [comment]
@kurtw yup its been updated and merged into flapjack. waiting on the push to Refresh
https://github.com/cfpb/cfgov-refresh/pull/1250

**Comment by kave** [comment]
@tnferrell could you close this if its no longer an issue
